sponsorship_compassion/model/child_compassion.py

```diff
@@ -64,11 +63,11 @@ def _get_unsponsored_since(self):
     @api.multi
     def _recompute_unsponsored(self):
         """ Useful for updating unset values """
-        self._store_set_values(['unsponsored_since'])
+        self.env['compassion.child']._store_set_values(['unsponsored_since'])

```

Comment:
This will recompute values for all children. Is there a way to call it only on modified children (self recordset) ?